tool/src/java/org/sakaiproject/gradebookng/tool/panels/AddGradeItemPanelContent.java

```diff
@@ -135,15 +147,31 @@ protected void onUpdate(AjaxRequestTarget target) {
 				//if extra credit, deselect and disable the 'extraCredit' checkbox
 				CategoryDefinition category = categoryMap.get(selected);
 				
-				if(category.isExtraCredit()) {
+				if(category != null && category.isExtraCredit()) {
 					extraCredit.setModelObject(false);
 					extraCredit.setEnabled(false);
 				} else {
 					extraCredit.setEnabled(true);
 				}
 				target.add(extraCredit);
+
+
+				if (businessService.categoriesAreEnabled()) {
```

Comment:
yeah sweet. I do like the other Event handler but I reckon for this case this is better :)